tools/__init__.py

```diff
@@ -48,6 +48,10 @@ def actionNeedRoot(action):
 
         tools_logging.init(args)
 
+        if os.environ.get("XDG_SESSION_TYPE")!="wayland":
```

Comment:
this likely will break using for example weston compositor

Reply:
Maybe?
```python
 if os.environ.get("XDG_SESSION_TYPE")=="x11":
```

Reply:
The problem is that weston does not set XDG_SESSION_TYPE

![image](https://user-images.githubusercontent.com/7735145/144631092-1744dbac-1cce-4f2f-ad6c-e8104fb1acb1.png)

Reply:
There seems to be another way to check if a wayland server is running and can be connected (how the apps check it - since gedit for ex starts in weston when it's there, even when launched outside)

It could be checked that way instead

Reply:
There is an environment variable called WAYLAND_DISPLAY if it is set that means that a wayland compositor is running, that should be used to check instead of XDG_SESSION_TYPE

Reply:
@mkg20001  
Could you check it?

Reply:
yes, that one works

so it should be

```suggestion
        if not os.environ.get("WAYLAND_DISPLAY"):
```

![image](https://user-images.githubusercontent.com/7735145/145695600-5c7b3758-5557-48c4-88e6-a152a9259e7b.png)